pyupgrade/_plugins/defauldict_lambda.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import ast
+import functools
+from typing import Any
+from typing import Iterable
+
+from tokenize_rt import Offset
+
+from pyupgrade._ast_helpers import ast_to_offset
+from pyupgrade._ast_helpers import is_name_attr
+from pyupgrade._data import register
+from pyupgrade._data import State
+from pyupgrade._data import TokenFunc
+from pyupgrade._token_helpers import find_and_replace_call
+
+_ITERABLE_BUILTINS = frozenset(('dict', 'list', 'set', 'tuple'))
+
+
+def _lambda_constant_replacement(value: Any) -> str | None:
```

Comment:
Should this be inlined in the next function ?

Reply:
yeah especially given the use of `Any` which really shouldn't be happening at all!